src/main/scala/com/cognite/sdk/scala/common/dataTypes.scala

```diff
@@ -12,6 +12,7 @@ import io.scalaland.chimney.dsl._
 final case class ItemsWithCursor[A](items: Seq[A], nextCursor: Option[String] = None)
 final case class Items[A](items: Seq[A])
 final case class ItemsWithIgnoreUnknownIds(items: Seq[CogniteId], ignoreUnknownIds: Boolean)
+final case class ItemsWithIgnoreUnknownIdsGeneric[A](items: Seq[A], ignoreUnknownIds: Boolean)
```

Comment:
Shouldn't we just make `ItemsWithIgnoreUnknownIds[A]` then? Do we really need both?